src/lib/stdout-manipulator.ts

```diff
@@ -54,7 +54,10 @@ function color(line: string, noClear: boolean = false): string {
   return line;
 }
 
-export function print(noColors: boolean, noClear: boolean, line: string): void {
+export function print(noColors: boolean, noClear: boolean, line: string, signalEmittedFiles: boolean): void {
+  const outputLine = noColors ? line : color(line, noClear);
```

Comment:
I think that the content of the function should be like this:
```
if (signalEmittedFiles && outputLine.startsWith('TSFILE:')) { 
  return; 
}
console.log(noColors ? line : color(line, noClear));
```

Reply:
@gilamran Definitively, I started to introduce `outputLine` but actually we don't need it (we need the raw line) it was fixed in the same commit 81c3159fed10c89b3bbb1a2d653f5f129237fa00 good catch